### PR TITLE
Fix passphrase preset: set passphrase for all key keygrips

### PR DESCRIPTION
**Pull request by yanndegat**
If a key contains multiple keygrips, then the passphrase preset
should be applied on each keygrip

could fix issues such as:
https://github.com/goreleaser/goreleaser-action/issues/201

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -50,14 +50,15 @@ async function run(): Promise<void> {
...
-      core.debug(`${keygrip}`);
-
-      core.info('🔓 Presetting passphrase');
-      await gpg.presetPassphrase(keygrip, process.env.PASSPHRASE).then(stdout => {
-        core.debug(stdout);
-      });
+      core.info('📌 Getting keygrips');
+      const keygrips = await gpg.getKeygrips(privateKey.fingerprint);
+
+      for (var i = 0; i < keygrips.length; i++) {
```
Comment (crazy-max) [inline src/main.ts:56]: Can you switch to an iterator here please?

```
for (let keygrip of await gpg.getKeygrips(privateKey.fingerprint)) {
```

File: __tests__/gpg.test.ts [hunk 2]
```diff
@@ -58,12 +58,12 @@ describe('gpg', () => {
...
-    it('returns the keygrip', async () => {
+  describe('getKeygrips', () => {
+    it('returns the keygrips', async () => {
       await gpg.importKey(userInfo.pgp);
-      await gpg.getKeygrip(userInfo.fingerprint).then(keygrip => {
-        console.log(keygrip);
-        expect(keygrip).toEqual(userInfo.keygrip);
+      await gpg.getKeygrips(userInfo.fingerprint).then(keygrips => {
+        console.log(keygrips);
+        expect(keygrips[0]).toEqual(userInfo.keygrip);
```
Comment (crazy-max) [inline __tests__/gpg.test.ts:66]: Add an assertion with the expected number of keygrips.

Also update `userInfo` def?

**Review by crazy-max — CHANGES_REQUESTED** [review]
@yanndegat Please follow and apply the [contributing guidelines](https://github.com/crazy-max/ghaction-import-gpg/blob/master/.github/CONTRIBUTING.md). Thanks for your contrib!

File: __tests__/gpg.test.ts (hunk 2 above)
Comment (yanndegat) [reply __tests__/gpg.test.ts:66]: hi @crazy-max 

what do you mean by updating userInfo def ?

Comment (crazy-max) [reply __tests__/gpg.test.ts:66]: Hi @yanndegat

Update the [userInfo object](https://github.com/crazy-max/ghaction-import-gpg/blob/master/__tests__/gpg.test.ts#L21). `keygrip` is now an array not a string.

Comment (yanndegat) [reply __tests__/gpg.test.ts]: updated. tell me if it's what you had in mind

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Pull request merged**